url-shortener/android/app/src/test/java/com/nubank/urlshortener/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.nubank.urlshortener
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
Do we need it?

Reply:
I recommend we keep this file because it will avoid the testing folder being removed and can help the candidates when creating a new test.